txstratum/cli.py

```diff
@@ -52,7 +53,8 @@ def execute(args: Namespace) -> None:
 
     backend = HathorClient(args.backend)
     manager = TxMiningManager(
-        backend=backend
+        backend=backend,
+        address=args.address,
```

Comment:
Should we validate the address string? I think in the future we should have a method on hathor-commons to validate depending on the network and then this would be easy to do.

Reply:
I agree. I'll make this change.

Reply:
Done!